tests/integration_tests/util.py

```diff
@@ -222,3 +349,63 @@ def get_feature_flag_value(client: "IntegrationInstance", key):
     if "NameError" in value:
         raise NameError(f"name '{key}' is not defined")
     return value
+
+
+def restart_cloud_init(c: "IntegrationInstance"):
```

Comment:
> Why is this needed?

If I remember correctly, this was created because `.restart()` didn't work on lxd vms. This was originally added back in a60c0845806baff72c7, and this PR only refactors it to be reusable (i.e. by two different tests that both use it).

Reply:
Although, now that I say that I can't reproduce the issue that I recall seeing before. I'll drop it.